Library/Homebrew/cmd/update-report.rb

```diff
@@ -115,7 +115,8 @@ def output_update_report
     install_core_tap_if_necessary
 
     updated = false
-    new_repository_version = nil
+    new_tag = nil
+    old_tag = nil
```

Comment:
Are you sure these are needed when they weren't before?

Reply:
https://github.com/Homebrew/brew/blob/88a5fac96752f17280e64f60926f58dff45990e0/Library/Homebrew/cmd/update-report.rb#L118

Reply:
`new_tag` needs to be declared because `return if new_tag.blank? || args.quiet?` can be encountered without `new_tag` being assigned. `old_tag = nil` is not needed and I can remove.